helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -350,13 +350,18 @@ agent:
       memory: 500Mi
 
   # To collect container logs and journald logs, it will run the agent as a root user.
-  # To run it as non root user, uncomment below `securityContext` options.
+  # To run it as non root user, change runAsUser and runAsGroup to non-zero value (e.g. 20000) and change runAsNonRoot to true.
```

Comment:
Also mention here that running as non-root user doesn't work in openshift cluster